.github/actions/setup/action.yml

```diff
@@ -0,0 +1,10 @@
+name: Setup
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
```

Comment:
This action caches global node_modules by default, when `cache` is present. I cleaned up the `widgets` version that had custom caching for readability and easier maintenance. I think we don't need a `node_modules` specific caching for smaller packages, Yarn will be fast anyway re-instantiating from global cache. 

This is open discussion!